{{cookiecutter.project_slug}}/pyproject.toml

```diff
@@ -114,7 +114,7 @@ coverage = {extras = ["toml"], version = "^5.0.4"}
 # style
 flake8 = "^3.7.9"
 pylint = "^2.4.4"
-mypy = "^0.770"
+mypy = "^0.800"
```

Comment:
I'd suggest using  syntax like `{% if int({{ cookiecutter.python_version }}) >= 3.7 %}` as 3.7 is still officially supported.

Reply:
I accidentally found this site quite useful. https://endoflife.date/python

Reply:
Good idea. Will update. The link is also interesting!

Reply:
Preliminary study shows version comparison with Jinja2 is difficult. I may dive into cookiecutter itself to see if version comparison function is supported. : (

Reply:
@tai271828 Is there any update on this one?

Reply:
No, QQ. I did not work on this anymore. It's in my wish list of task. Feel free to close the pull request.

Reply:
No worries 🙂 Let's make it a draft one so that I know I don't need to review it at this moment.

Reply:
Sounds good! Draft is a good idea so we could keep the track.

Yeah I have had dug into this issue and found it is a rabbit hole : (